yasnippet.el

```diff
@@ -1083,7 +1083,7 @@ Has the following fields:
         (maphash #'(lambda (k v)
                      (let ((template (gethash name v)))
                        (when (and template
-                                  (eq uuid (yas--template-uuid template)))
+                                  (equal uuid (yas--template-uuid template)))
```

Comment:
Locally, makes perfect sense to me, since UUID's are strings, not symbols or numbers.

In a broader note, though, I'm quite embarassed of having come up (actually I copied it from textmate) this UUID-based CRUD scheme when Lisp's symbols would have worked much much nicer.